layers/+spacemacs/spacemacs-evil/packages.el

```diff
@@ -155,6 +156,12 @@
       (define-key evil-visual-state-map evil-exchange-cancel-key
                   'evil-exchange-cancel))))
 
+(defun spacemacs-evil/init-evil-fringe-mark ()
+  (use-package evil-fringe-mark
+    :config
+    (setq-default evil-fringe-mark-show-special t)
```

Comment:
I think it would be better to set options in the `:init` or `:custom` block. It makes them easier to override by users when the package is deferred.